pkgs/tools/misc/1config/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, stdenv
+, autoPatchelfHook
+, fetchurl
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  name = "1config";
+  version = "0.21.0";
+
+  buildInputs = [ stdenv.cc.cc.lib zlib ];
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  src = fetchurl {
+    url = "https://github.com/BrunoBonacci/1config/releases/download/0.21.0/1cfg-Linux";
+    sha256 = "0rynj8q903i3gypgmyvv43vk0hfz3q7bk2l65pmmk6fhyg9r2r9z";
+    executable = true;
+  };
```

Comment:
Fetching a binary when building from source is possible, is generally frowned upon

Reply:
Ah sorry, I'll try to rework it to build the binaries instead

Reply:
@legendofmiracles I've had a go at building the binaries, but I couldn't get it working unfortunately. The issue is that the build uses `leiningen` to download packages outside of nix, which AFAIK isn't allowed (or at least I couldn't get it working). In theory I could try to generate a `pom.xml` and use `mvn2nix`, but to be honest I'm still pretty new to nix and I think it would take me too long to get that working.

I think the options at this point are either: close the PR and I'll have another go when I have more time and experience, or merge it as-is. Let me know what you think would be best, or I'm happy to try another approach if I've missed something :slightly_smiling_face:.